src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Is your native app installed? getInstalledRelatedApps() will tell you!
+subhead: The getInstalledRelatedApps() method allows your web app to check whether your native app is installed on a user's device, and vice versa.
+authors:
+  - petelepage
+description: As the capability gap between web and native gets smaller, it gets easier to offer the same experience for both web and native users. This may lead to cases where users have both web and native versions of the same app installed on the same device. Apps should be able to detect this situation. The getInstalledRelatedApps() API is a new web platform API that allows your web app to check to see if your native app is installed on the users device, and vice versa.
+date: 2018-12-20
+updated: 2019-10-11
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+hero: hero.jpg
+alt: mobile device with app panel open
+# draft: true
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/855683929200394241
+---
+
+{% Aside %}
+  We're currently working on this API as part of the new
+  [capabilities project](https://developers.google.com/web/updates/capabilities).
+  Starting in Chrome 73, it is available as an [origin trial](#ot) on Android.
+  This post will be updated as the API evolves.
+{% endAside %}
+
+## What is the `getInstalledRelatedApps()` API? {: #what }
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="getinstalled-cropped.jpg" class="w-screenshot" width="550">
+  <figcaption class="w-figcaption">
+    A web app using `getInstalledRelatedApps()` to determine if it's
+    related native app is already installed.
+  </figcaption>
+</figure>
+
+As the capability gap between web and native gets smaller, it gets easier to
+offer the same experience for both web and native users. This may lead to cases
+where users have both web and native versions of the same app installed on the
+same device. Apps should be able to detect this situation.
+
+The [`getInstalledRelatedApps()`][spec] method allows your web app to check if
```

Comment:
We've been starting to use `rel=noopener` on external links. I didn't convert all the links in this post but wanted to flag it for future posts.

Reply:
Can we look into how to do that in the markdown parser instead? I won't remember to do that all the time, and I'd rather use markdown consistently.

Reply:
Defer to @robdodson on this.

Reply:
I've filed #1670 for this.

I'd offer to do it, but I'm not exactly sure where it would go in the codebase.